language/move-prover/abigen/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
```

Comment:
Not sure if there are crate restrictions at play here (I can't see any?), but it seems a bit odd to have `abigen` under move-prover. Could this instead be a crate in the top level `language`, or possibly a crate under `stdlib`?

Reply:
I followed the example of `docgen` (which is also called by stdlib). We may want to figure out a better place for both crates.

Reply:
Agreed. I don't view this placement as a blocker for this PR -- better to get it in and being used and then we can move it around if need be.